### PR TITLE
Feature/country shape

### PR DESCRIPTION
**Pull request by agnlez**

### CODE REVIEW

File: _sass/components/_c-section-title.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 .c-section-title {
   position: relative;
-  padding-bottom: rem(50px);
+  // padding-bottom: rem(50px);
```
Comment (dhakelila) [inline _sass/components/_c-section-title.scss:3]: Be extremely careful when doing changes in this general elements.  You might be affecting more elements.
Be sure you are not crashing another elements and if so, make and exception (&.-my-awesome-exception)

File: _sass/layout/_l-banner.scss [hunk 2]
```diff
@@ -45,8 +45,16 @@ $backgrounds: 'promoHome', 'promoInteriores-About', 'promoInteriores-CompareCoun
 
     > .wrap {
       display: flex;
-      justify-content: center;
-      flex-direction: column;
+      justify-content: space-between;
+      flex-direction: row;
```
Comment (dhakelila) [inline _sass/layout/_l-banner.scss]: Be extremely careful when doing changes in this general elements.  You might be affecting more elements.
Be sure you are not crashing another elements and if so, make and exception (&.-my-awesome-exception)

Hint: Check one index page.

File: _sass/layout/_l-banner.scss [hunk 3]
```diff
@@ -45,8 +45,16 @@ $backgrounds: 'promoHome', 'promoInteriores-About', 'promoInteriores-CompareCoun
...
+    }
+
+    > .wrap > .js--country-silhouette {
+
+      > svg {
+        fill: $beige;
+        opacity: .3;
+      }
```
Comment (dhakelila) [inline _sass/layout/_l-banner.scss:65]: I know this os not too much code, but I think you should make a new module to fit the scss structure

**Comment by dhakelila** [comment]
Please, follow naming convention in this project. I don't mean "countryHelper.js" is wrong but we are namming as "country_helper.js" instead.

Also, I'd rather add this view to common folder instead of library, but we can talk about this if you want. 
Again, I don't mean is bad, but is not following project convention.

**Comment by agnlez** [comment]
Yep, you are right. I just moved all helpers (by now, 2) to a new folder called helpers :) Check the name of the helpers out, please. I think call "helpers.js" a file that only has Handlebars helpers is wrongs so i rename it to handlebars.js inside helpers folder. I did the same for countryHelper.js, now is called country_drawer.js. More explicit :)

**Comment by dhakelila** [comment]
Looks good to me! 
Only, I'd name handlebars_helpers.js, car handlebars.js is the library...

**Comment by agnlez** [comment]
Well, you are inside the helpers folder, it should be obvious that's not the library. I think it's too redundant call a file whatever_helpers when you are already in helpers. But as you want!

**Pull request merged**